app/controllers/custom_attributes_controller.rb

```diff
@@ -35,35 +35,36 @@ def choice
   end
 
   private
-  def check_type_param
-    redirect_to root_path if params[:type].blank?
-  end
 
-  def update_existing_attributes(params)
-    attributes = CustomAttribute.attributes_for(current_user.company, params[:type])
+    def check_type_param
```

Comment:
Do you indent these manually or is your editor auto indenting methods after `private`/`protected`? Also on that note, which editor do you use? I'm using Emacs, and it does _not_ indent.